utils/atheepmgr/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2021 Sergey Ryazanov <ryazanov.s.a@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atheepmgr
+PKG_VERSION:=2.1.1
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rsa9000/$(PKG_NAME).git
+PKG_MIRROR_HASH:=2c0c9705b784f736d1fb32777849d1cc7cefeee0137512f0624f5045ebf6fef6
+
+PKG_LICENSE:=ISC
+PKG_MAINTAINER:=Sergey Ryazanov <ryazanov.s.a@gmail.com>
+
+PKG_CONFIG_DEPENDS:=\
+	CONFIG_ATHEEPMGR_DANGEROUS_FUNCTIONS \
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/atheepmgr
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=EEPROM/boarddata management utility for Atheros WLAN chips
+  MENU:=1
+endef
+
+define Package/atheepmgr/description
+This utility is intended to investigate the content of Atheros WLAN
+chips EEPROM (OTP, boarddata, ART, etc.). It supports data extraction,
+unpacking, parsing, dumping in a text form, saving binary data for
+further analysis, as well as quick editing of some basic fields (e.g.
+MAC address, etc.).
+
+The utility supports both data access methods: direct EEPROM (OTP)
+access (via the chip) and offline binary dumps handling.
+
```

Comment:
why an empty line?

Reply:
Empty line before block to separate paragraphs. Line after was added to separate maintainer line, which will be added automatically from the main help text.

Reply:
Could u be more specific? I never had this issue and I never saw someone adding a free line before `endef`.

Reply:
Sure. Without trailing empty line, _make menuconfig_ help text (when you type '?') will looks like:
> The utility supports both data access methods: direct EEPROM (OTP)
> access (via the chip) and offline binary dumps handling.
> Sergey Ryazanov \<user@example.com\>

With this separator line, help text become:
> The utility supports both data access methods: direct EEPROM (OTP)
> access (via the chip) and offline binary dumps handling.
>
> Sergey Ryazanov \<user@example.com\>

Notice the empty line between end of text and the maintainer line.

I noticed this situation when check package metadata before submission and intentionally add the empty line at the end of description to make help text readable.

Reply:
Thanks. Maybe this is more a general style issue. ;)
I would prefer your style in every helptext.

Reply:
I never saw any rules regarding helptext, so I will close that here as resolved. Sometimes people use tabs or spaces if they have a longer helptext.

Reply:
This happened due to the following package info generation code (include/package-dumpinfo.mk, DumpInfo/Package macro, lines 47-49):
> )Description: $(if $(Package/$(1)/description),$(Package/$(1)/description),$(TITLE))
> $(if $(URL),$(URL)
> )$(MAINTAINER)

To add the separator line, code should look like this (not tested):
> )Description: $(if $(Package/$(1)/description),$(Package/$(1)/description),$(TITLE))
> $(if $(URL),$(URL)
> )$(if $(MAINTAINER),
> $(MAINTAINER))

But this proposal do not account case, when a packages description is empty. In case of empty description we should not insert the separator line before a maintainer info.